wiring/inc/spark_wiring_cloud.h

```diff
@@ -397,6 +400,84 @@ class CloudClass {
         const String* s = (const String*)var;
         return s->c_str();
     }
+
+    // This method takes an argument of any callable type that requires no arguments and returns
+    // a value of one of the supported variable types
+    template<typename T, typename CloudVariableType<typename std::result_of<T()>::type>::ValueType* = nullptr>
```

Comment:
Perhaps fix for functions returning `char*` instead of `const char*`? An easy mistake to make.

Reply:
Fixed